.artist_resources/generic religion icons list.txt

```diff
@@ -0,0 +1,114 @@
+## Vanilla
```

Comment:
Mmmm. So to avoid such situations you shouldn't forget to update the destination branch of `physical-perks` and then merge the destination branch to this one.

Reply:
Lmao, resolved since this was your fault